tests/test_translation.sh

```diff
@@ -11,10 +11,10 @@ for model in "Helsinki-NLP/opus-mt-en-de" "sshleifer/tiny-mbart" ; do
 
     if [[ $model == Helsinki-NLP* ]] ; then
       base_model="marian"
-      expected_result='{"bleu": 95.37006954777766}'
+      expected_result='{"sacrebleu": 88.04086004116694}'
```

Comment:
Why is the metric called sacrebleu now? sacrebleu is just the implementation, the metric is still bleu.

Reply:
For backward compatibility, I didn't wanna change the definition of "our" bleu score. The only difference between `blue` and `sacrebleu` is that `sacrebleu` does not do lowercasing so it's appropriate for NLG or other tasks that preserev casing. Otherwise, their implementation is the same. If you all prefer, I can change "bleu" to not lowercase and remove "sacrebleu". Just beware that previous bleu scores on non-almond datasets will be lower.

Reply:
Maybe "casedbleu" then? "sacrebleu" is not a meaningful metric name, it doesn't tell you what's the difference with normal bleu

Reply:
okay